grails-app/views/video/_report.gson

```diff
@@ -7,13 +8,14 @@ model {
 json g.render(report,[includes: []]) {
   id report.id
   dateCreated report.dateCreated
+  lstUpdated report.lastUpdated
   resolved report.resolved
   userId report.createdBy.id
   userName report.createdBy.username
-
+  if (report.video instanceof Episode) {
+    Episode episode = report.video as Episode
+    season episode.season_number
+  }
```

Comment:
Noted. I was often not sure in these cases because the commit doesn't directly send the episode data but rather lets you do it. In this case it still kinda makes sense but I believe there were cases where the sentence didn't make sense to me without 'let you'. But if this is okay, then I'll just leave the let you out from now on.

Reply:
Makes sense. As long as you keep it in mind itll be fine :) if you make a conscious decision to include "let you" from now on, then I'll trust that you put good thought into it :)